src/solver/conv_direct_naive_conv.cpp

```diff
@@ -244,5 +256,49 @@ bool ConvDirectNaiveConvIsApplicableByKernelType(const ExecutionContext& ctx,
     return true;
 }
 
+// figure out the index of C (channel) stride so we can expand it into
+// (G, C_per_group). Return value G_stride_idx is the position of G stride
+// in the stride vector, such that the (G_stride_idx - 1) is the index that
+// contains C's stride as a multiplying factor
```

Comment:
[R] Doxygen & punctuation
```suggestion
/// Figure out the index of C (channel) stride so we can expand it into
/// (G, C_per_group). Return value G_stride_idx is the position of G stride
/// in the stride vector, such that the (G_stride_idx - 1) is the index that
/// contains C's stride as a multiplying factor.
```

Reply:
[resolved]